Main.cpp

```diff
@@ -421,26 +423,24 @@ bool PlayIntroTitle() {
     BITMAP **apStarLargeBitmaps = starLargeFile.GetAsAnimation(starLargeBitmapCount);
     BITMAP **apStarHugeBitmaps = starHugeFile.GetAsAnimation(starHugeBitmapCount);
     Star *aStars = new Star[starCount];
```

Comment:
You can remove the a from here too, pretty clear it's an array of stars. So, just call it `stars` or, if you really want, `starsArray`.
There's definitely also other names in this stuff you can clean up if you want, but don't feel obliged to go too nuts.